docs/ReleaseNotes.md

```diff
@@ -1,6 +1,32 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 3.1.2
+Acuminator 3.1.2 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Enhancements
+In Acuminator 3.1.2, the following enhancements have been implemented:
+ - The [PX1096](diagnostics/PX1096.md) diagnostic has been added to check signature of methods with the `PXOverride` attribute.
```

Comment:
I would rephrase this a bit, it's not just a some kind of generic unspecified check of PXOverrides. It's a check that in the base graph and base graph extensions there exists a base method that is compatible with the PXOverride method. This means that the base method has the same name, its accessibility public, protected or protected internal, and it has a compatible parameters.  More about this here:
- https://help.acumatica.com/(W(147))/Help?ScreenId=ShowWiki&pageid=6fa2a444-17b4-42f9-9e6a-64e85167626a
- https://help.acumatica.com/(W(146))/Help?ScreenId=ShowWiki&pageid=635c830e-4617-4d5c-9fa5-035952311aa9